tools/misc/tpm2_checkquote.c

```diff
@@ -304,6 +342,69 @@ static tool_rc init(void) {
         }
     }
 
+    if (ctx.flags.eventlog) {
+        tpm2_eventlog_ctx_t eventlog_ctx = {};
+        if (!eventlog_from_file(&eventlog_ctx, ctx.eventlog_path)) {
```

Comment:
dont hide functions in conditionals, set an intermediate value and check that in the if statement. The only exceptions are if it makes the code easier.